docs/src/production_readiness.md

```diff
@@ -0,0 +1,93 @@
+# Production Readiness Guide
+
+Running a production grade PostgreSQL system requires effort and preparation. If
+you're coming to CloudNativePG with PostgreSQL experience, or with other
+relational database  experience, you should already know much of what you need
+to have a production-ready CNPG system, with the additional benefits brought by
+the Kubernetes ecosystem.
+
+Whether a novice or an expert, please review this document to make
+sure you're production ready with CloudNativePG. The following sections should
+be considered a minimum.
+
+## Set up a regular backup regime
+
+You should not have a database in production without taking regular backups.
+CloudNativePG can help with scheduled backups, using object stores or volume
+snapshots.
+
+See [CNPG backup documentation](https://cloudnative-pg.io/documentation/current/backup/).
+
+## Regularly test restore from backup
+
+Having backups is essential. It is equally essential to test those backups.
+CloudNativePG  makes it very easy to spin up a cluster from a backup. You should
+regularly check backups are recent and in good condition.
+
+See [CNPG recovery documentation](https://cloudnative-pg.io/documentation/current/recovery/).
+
+## Set up monitoring
+
+It goes without saying, a production database should be monitored, not only for
+uptime, load etc., but for space. In addition, it is vitally important to ensure
+that WAL files don't pile up and potentially fill your `PGDATA` volume, or your
+dedicated WAL storage volume. The CloudNativePG Quickstart guide includes
+instructions on setting up a Grafana / Prometheus monitoring system. Prometheus
```

Comment:
is a semicolon too elitist?

Reply:
speaking about line 35 at the end

Reply:
Not too elitist, no. But, is that right?
I've thought of the semicolon as an "incomplete". The second sentence completes the first.
In this case, the second sentence feels independent.
But if you feel the need for a semicolon, I trust you on it.